### PR TITLE
pre-warm cache of tts

### PR DESCRIPTION
**Pull request by benadida**
## Overview

Make use of the new kiosk-browser API that caches spoken utterances, to warm the text-to-speech cache.

## Testing Plan

Tested manually

## Checklist

~- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.~

### CODE REVIEW

File: frontends/bmd/src/pages/admin_screen.tsx [hunk 1]
```diff
@@ -71,6 +75,100 @@ export function AdminScreen({
     fetchElection();
   }
 
+  useEffect(() => {
```
Comment (benadida) [inline frontends/bmd/src/pages/admin_screen.tsx:78]: this brute-forcing of strings to pre-warm the cache is... not my favorite approach. But, it has the virtue of being simple, not super likely to fail, and if it fails, the worst outcome is some strings aren't cached and will be when they're later spoken.

File: frontends/bmd/src/app_root.tsx [hunk 2]
```diff
@@ -535,8 +538,98 @@ export function AppRoot({
     /* istanbul ignore else */
     if (newElectionDefinition) {
       updateElectionDefinition(newElectionDefinition);
+
+      void (async () => {
```
Comment (benadida) [inline frontends/bmd/src/app_root.tsx:542]: this isn't the most resilient approach, as it assumes a lot about what the subsequent strings are. But, for a first pass, it has the benefit of being unlikely to fail in a way that breaks the app. If it fails to cache, worst case is that the utterances are delayed on first use.

Comment (benadida) [reply frontends/bmd/src/app_root.tsx:542]: putting this here in the app_root also means that loading the election is not in any way delayed. The utterances are generated in the background, even if the election manager card is removed.

Comment (carolinemodic) [reply frontends/bmd/src/app_root.tsx:542]: So if we miss a string here the failure mode is it just takes a while to be said?

**Review by carolinemodic — APPROVED** [review]

**Pull request merged**